CHANGELOG.rst

```diff
@@ -17,14 +17,20 @@
   is specified on the command line together with the ``--pyargs``
   option.  Thanks to `@taschini`_ for the PR (`#1597`_).
 
-*
+* Raise helpful failure message, when requesting parametrized fixture at runtime,
+  e.g. with ``request.getfuncargvalue``. BACKWARD INCOMPAT: Previously these params
```

Comment:
As we discussed during the planning today, we would like to change that to `getfixturevalue` (keeping the older name only for backward compatibility). Would you like to do this in this PR? If not, would you please open a new issue? Thanks! :grin: